src/components/DataViewerCell.vue

```diff
@@ -0,0 +1,32 @@
+<template functional>
+  <td
+    :class="{'data-viewer-cell': true, 'data-viewer-cell--active': props.isSelected }"
+  >{{ props.value }}</td>
+</template>
+<script lang="ts">
+import Vue from 'vue';
+import { Component, Prop } from 'vue-property-decorator';
+
+/**
+ * @name DataViewerCell
+ * @description A cell display in a DataViewer
+ * @param {boolean} isSelected - Wether the cell is selected or is in a selected column
+ * @param {string} value - The cell's value
+ */
+@Component({
+  name: 'data-viewer-cell',
+})
+export default class DataViewerCell extends Vue {
+  @Prop({
+    default: () => false,
+  })
+  isSelected!: boolean;
+
+  @Prop({
+    default: () => '',
+  })
+  value!: string;
```

Comment:
Is this always a string?
If values are not formatted, this can be pretty much anything like Number, null/undefined or even Object (for _id fields sent by Mongo)

Reply:
I choose to consider it like as it's "just" the display in the table. It seems that specifying string here cast automatically cf. fake_dataset. But open to discussion

Reply:
I don't think adding a type in TS does an automatic casting.
But using `{{ value }}` in the template, I think vue does the casting automatically.
Still, the TS compiler will fail if we pass a dataset which have different value types.